jiant/utils/options.py

```diff
@@ -0,0 +1,40 @@
+"""
+Functions for parsing configs.
+"""
+import torch
+
+
+def parse_task_list_arg(task_list):
+    """Parse task list argument into a list of task names."""
+    task_names = []
+    for task_name in task_list.split(","):
+        if task_name == "glue":
+            task_names.extend(ALL_GLUE_TASKS)
+        elif task_name == "superglue":
+            task_names.extend(ALL_SUPERGLUE_TASKS)
+        elif task_name == "none" or task_name == "":
+            continue
+        else:
+            task_names.append(task_name)
+    return task_names
+
+
+def parse_cuda_list_arg(cuda_list):
+    """
+    Parse cuda_list settings
+    """
+    result_cuda = []
+    if cuda_list == "auto":
+        cuda_list = list(range(torch.cuda.device_count()))
+        return cuda_list
+    elif isinstance(cuda_list, int):
+        return cuda_list
+    elif "," in cuda_list:
+        return [int(d) for d in cuda_list.split(",")]
+    else:
+        raise ValueError(
+            "Your cuda settings do not match any of the possibilities in defaults.conf"
+        )
+    if len(result_cuda) == 1:
```

Comment:
we shouldn't hit this ever , I think

Reply:
We will hit this for if someone puts in "auto" and there's only 1 GPU in the device. I'll move this up.